add benchmarks into readme

davidbuniat opened the pull request:
## 🚀 🚀 Pull Request

### Checklist:

- [ ]  [My code follows the style guidelines of this project](https://www.notion.so/activeloop/Engineering-Guidelines-d6e502306d0e4133a8ca507516d1baab) and the [Contributing document](https://github.com/activeloopai/Hub/blob/release/2.0/CONTRIBUTING.md)
- [ ]  I have commented my code, particularly in hard-to-understand areas
- [ ]  I have kept the `coverage-rate` up
- [ ]  I have performed a self-review of my own code and resolved any problems
- [ ]  I have checked to ensure there aren't any other open [Pull Requests](https://github.com/activeloopai/Hub/pulls) for the same change
- [ ]  I have described and made corresponding changes to the relevant documentation
- [ ]  New and existing unit tests pass locally with my changes


### Changes

Adding readme benchmarks

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/activeloopai/deeplake?pullRequest=2100) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/activeloopai/deeplake?pullRequest=2100) before we can accept your contribution.<br/><hr/>**david.buniatyan@gmail.com** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/activeloopai/deeplake?pullRequest=2100) it.</sub>

mikayelh commented:
Unsure what is happening to the gif, are we discarding it? O/w seems great. Second graph could benefit from Hambardzumyan et al citation (to bring forth the CIDR publication)

davidbuniat commented:
> Unsure what is happening to the gif, are we discarding it? O/w seems great. Second graph could benefit from Hambardzumyan et al citation (to bring forth the CIDR publication)

not sure which second paragraph, added Hambardzumyan et al citation. Gif href link was wrong, fixed it with pointing to app.activeloop.ai coco dataset. Let me know if there is another link you thought needs to be but current one is not working.

mikayelh commented:
@davidbuniat the href link isn't wrong, this is intended, please leave as is. www.linkpicture.com/view.php?img=LPic61b13e5c1c539681810493
